katello_rapidnode.py

```diff
@@ -9,24 +9,21 @@
 from configparser import ConfigParser
 from locale import getdefaultlocale
 from termcolor import colored
-import os
 import paramiko
 
-REPO_FILE = 'myrepofile.repo'
 CONFIG = ConfigParser()
 CONFIG.read('katello_rapidnode.ini')
 PARENT = CONFIG['servers']['parent']
 
 
 def main():
+    "Main function to setup capsule(s)"
```

Comment:
Use triple quotes when creating a docstring.

    """Main function to set up capsule(s)."""

Also, there's no great need to write "Main function", as this is by definition a function called "main." It's just as informative to write something like this:

    """Set up one or more capsules."""